lib/Backend/Lower.cpp

```diff
@@ -2009,6 +2009,13 @@ Lowerer::LowerRange(IR::Instr *instrStart, IR::Instr *instrEnd, bool defaultDoFa
                         this->LowerBrCMem(instr, IR::HelperOp_Equal, false);
                     }
                 }
+                else if (this->GenerateFastBrEqBoolInt(instr->AsBranchInstr(), &needHelper))
```

Comment:
>GenerateFastBrEqBoolInt [](start = 31, length = 23)

Should we also have this for CmEq_A and CmNeq_A as most of the code is reusable?